packages/pwa-kit-create-app/assets/templates/extension-starter/src/setup-app.ts.hbs

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) 2025, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+// TODO: Brian test, remove this comment
+
```

Comment:
This is a lot of maintenance overhead in terms of having to now edit 2 sources when wanting to make changes to the setup app and setup server. 

Are we sure we want to take that leap?

Reply:
I had the same thought. I think the reason why we keep the original files on top of the hbs files is because we want to keep the `@salesforce/extension-starter` functional so we can install it directly in the typescript-minimal for testing.

https://github.com/SalesforceCommerceCloud/pwa-kit/blob/feature/%40W-18272699%40/extension-folder-class-names/packages/extension-starter/src/setup-server.ts

Maybe we can automate update or combine files?